completions/fio

```diff
@@ -24,13 +31,13 @@ _fio()
             return
             ;;
         --cmdhelp)
-            # TODO more commands?
-            COMPREPLY=($(compgen -W "all" -- "$cur"))
+            ret=($("$1" --cmdhelp=all 2>/dev/null | awk '{print $1}'))
+            COMPREPLY=($(compgen -W '${ret[@]} all' -- "$cur"))
```

Comment:
- Avoid the `nounset` error in bash 4.2 and 4.3 for `${ret[@]}` in case the command fails.
- Avoid redundant word splitting by quoting `"..."`(because the necessary word splitting should have been performed in the previous line).

```suggestion
            ret=($("$1" --cmdhelp=all 2>/dev/null | awk '{print $1}') all)
            COMPREPLY=($(compgen -W '"${ret[@]}"' -- "$cur"))
```

Reply:
Ah, clever way to avoid that nounset ret :+1: